homepage: https://github.com/jcristovao/enclosed-exceptions
changelog-type: ''
hash: 727dda45f3e2989d845ea7d645a22a72f3226b0ab5cc66da8276f8745a099375
test-bench-deps:
  stm: -any
  base: -any
  hspec: ! '>=1.3'
  monad-control: -any
  async: ! '>=2.0'
  lifted-base: ! '>=0.2'
  transformers-base: -any
  transformers: -any
  deepseq: -any
  QuickCheck: -any
maintainer: jmacristovao@gmail.com, michael@snoyman.com
synopsis: Catching all exceptions from within an enclosed computation
changelog: ''
basic-deps:
  base: ! '>=4 && <5'
  monad-control: -any
  lifted-base: ! '>=0.2'
  transformers-base: -any
  transformers: -any
  deepseq: -any
all-versions:
- '1.0.0'
- '1.0.0.1'
- '1.0.0.2'
- '1.0.1'
- '1.0.1.1'
- '1.0.2'
author: Michael Snoyman, João Cristóvão
latest: '1.0.2'
description-type: haddock
description: ! 'Catching all exceptions raised within an enclosed computation,

  while remaining responsive to (external) asynchronous exceptions.

  For more information on the technique, please see:

  <https://www.fpcomplete.com/user/snoyberg/general-haskell/exceptions/catching-all-exceptions>'
license-name: MIT
